gradle.properties

```diff
@@ -7,12 +7,15 @@ dirConfig = config
 dirLib = lib
 dirQuality = staticAnalysis
 
-org.gradle.jvmargs=-Xms256m -Xmx2048m
+# org.gradle.jvmargs=-Xms256m -Xmx2048m
+org.gradle.jvmargs=-Xms2560m -Xmx8096m
 
 # analysis properties
 checkstyleVersion = 8.35
 ## Updating to 6.37.0 lead to a timeout when publishing -- update when the bug in pmd is fixed
 pmdVersion = 6.25.0
+#pmdVersion = 7.0.0-rc4
```

Comment:
The commented pmdVersion doesn't make sense

Reply:
7.0.0-rc4 is the latest version. I had issues with all 6.x pmd versions which appear and disappear at random. However, I can remove this.